blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -12,37 +12,40 @@ import * as pushEventUtil from './util/push-event-util';
 const { object, array, func, node, string } = PropTypes;
 
 class OrganizationPipelines extends Component {
-
-    // FIXME: IMO the following should be dropped
+    constructor(props) {
+        super(props);
+        this.state = {};
```

Comment:
You can drop the whole constructor, since you do not set anything on the state as default.